src/storage.ts

```diff
@@ -371,6 +374,13 @@ export class Storage extends Service {
    * @property {boolean} [autoRetry=true] Automatically retry requests if the
    *     response is related to rate limits or certain intermittent server
    * errors. We will exponentially backoff subsequent requests by default.
+   * @property {number} [backoffMultiplier = 2] Dictates the rate of exponential backoff.
+   * @property {number} [retryDeadline = 600] The length of time to keep retrying in seconds.
+   *     The last sleep period will be shortened as necessary, so that the last retry runs
+   *     at deadline (and not considerably beyond it).
+   * @property {number} [maxRetryTimeout = 64] The maximum time to delay in seconds.
```

Comment:
I wouldn't call this a timeout exactly I think-- I've heard the term "envelope" or "window" for this (spacing between retries).

Reply:
changed this to `totalTimeout` to match the retry customization variable of the google gax library

Reply:
I was confused but it looks like you actually used `MaxRetryDelay` from there for this param, not `TotalTimeout`

Reply:
whoops yeah my previous naming was super confusing. does it look right to you now?

Reply:
Yep much better!

Reply:
Yay thanks for the feedback!